plugins/community/infobloxopen-gorm/v1.1.2/buf.plugin.yaml

```diff
@@ -0,0 +1,7 @@
+version: v1
+name: buf.build/community/infobloxopen-gorm
+plugin_version: v1.1.2
+source_url: https://github.com/infobloxopen/protoc-gen-gorm
+description: Generate GORM models and APIs for simple object persistence tasks.
+spdx_license_id: Apache-2.0
+license_url: https://github.com/infobloxopen/protoc-gen-gorm/blob/v1.1.2/LICENSE
```

Comment:
Can you add an `output_languages` section too?

```yaml
output_languages:
  - go
```

Reply:
Added this key